modules/demo-magento/cypress/integration/blogCategory.js

```diff
@@ -1,20 +1,21 @@
-import BlogCategory from '~/cypress/support/pageObjects/BlogCategory'
+import getBlogTitle from '~/cypress/support/pageObjects/blogCategory/getBlogTitle'
+import getBlogPostTiles from '~/cypress/support/pageObjects/blogCategory/getBlogPostTiles'
+import visitRandom from '~/cypress/support/pageObjects/blogCategory/visitRandom'
 
 describe('Blog category', () => {
-  /** @type {BlogCategory} */
-  let blogCategory = null
-
   beforeEach(() => {
-    blogCategory = new BlogCategory()
-
-    blogCategory.visitRandom()
+    visitRandom().as('blogCategory')
```

Comment:
I know that visitRandom is in the blogCategory directory, so that implies it will visit random blog category, but from this line alone `visitRandom()` is not descriptive enough.